extensions/wikia/Helios/Helios.setup.php

```diff
@@ -35,3 +35,4 @@
 
 $wgHooks['UserSaveSettings'][] = 'Wikia\\Helios\\User::onUserSave';
 $wgHooks['UserSaveOptions'][] = 'Wikia\\Helios\\User::onUserSave';
+$wgHooks['UserLogout'][] = 'Wikia\\Helios\\User::clearAccessTokenCookie';
```

Comment:
Is `Wikia\\Helios\\User::clearAccessTokenCookie` method called only via this hook? if so, let's rename it to `onUserLogout` to match the methods above.